### PR TITLE
Fixes #53174 - Fixed cyborg advanced mop being available for all cyborgs modules

### LINKED ISSUE #53174: Cyborg advanced mops - applicable to non jani'borgs
## Reproduction:

1). Obtain a 'janitor cyborg advanced mop'. 
`/obj/item/borg/upgrade/amop`

2). Apply this upgrade board to any **NON** janitor cyborg module.

3). The upgrade board is rejected by the **NON** janitor cyborg module and dropped to the ground. 
`module_flags = BORG_MODULE_JANITOR`

4). _**The non janitor cyborg will obtain the advanced mop upgrade.**_

5). The board is now able to be used on other non janitor cyborgs to provide them with advanced mops.

### PR DESCRIPTION
**Pull request by Donkie**
## About The Pull Request
Fixes https://github.com/tgstation/tgstation/issues/53174

Codewriter had simply missed indenting the code one step so the module-giving code was being run despite the parent saying no-no

## Why It's Good For The Game
making the game better, one bug at a time

## Changelog
:cl:
fix: Fixed cyborg advanced mop being available for all cyborgs modules
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
Could you add mediborgs to the list of borg modules that can receive this upgrade? It'd be nice for them to gain the abiilty to clean medbay once they receive upgrades later in the round, and dragomagol expressed an enjoyment of having an advanced mop as a mediborg in the thread of the issue report about this.

**Comment by Donkie** [comment]
While I agree with you that it's nice to make cleaning supplies more accessible, this PR is merely a fix for that issue, I don't want to introduce anything new in it

**Comment by zxaber** [comment]
Sad day.

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -223,9 +223,9 @@
 		for(var/obj/item/mop/cyborg/M in R.module.modules)
 			R.module.remove_module(M, TRUE)
 
-	var/obj/item/mop/advanced/cyborg/A = new /obj/item/mop/advanced/cyborg(R.module)
-	R.module.basic_modules += A
-	R.module.add_module(A, FALSE, TRUE)
+		var/obj/item/mop/advanced/cyborg/A = new /obj/item/mop/advanced/cyborg(R.module)
+		R.module.basic_modules += A
+		R.module.add_module(A, FALSE, TRUE)
```
Comment (Jared-Fogle) [inline code/game/objects/items/robot/robot_upgrades.dm:228]:
```suggestion
		var/obj/item/mop/advanced/cyborg/mop = new /obj/item/mop/advanced/cyborg(R.module)
		R.module.basic_modules += mop
		R.module.add_module(mop, FALSE, TRUE)
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Review by Mothblocks — APPROVED** [review]

**Comment by RaveRadbury** [comment]
This isn't a bug, this was intended.

**Pull request merged**